### PR TITLE
install sdk-config.h

### LINKED ISSUE #1272: cmake install misses a header
**Describe your environment**
Ubuntu, wsl

**Steps to reproduce**
Install steps,
  mkdir build && cd build
  cmake .. -DBUILD_SHARED_LIBS=ON -DWITH_JAEGER=ON
  cmake --build . --target all
  ctest
  cmake --install . --config Release --prefix ../.

**What is the expected behavior?**
The include directory should be as per cmake OPENTELEMETRY_CPP_INCLUDE_DIRS by default, and it should contain all relevant header files.

**What is the actual behavior?**
If --prefix not given it installs in same folder.

header "opentelemetry/sdk_config.h" is not copied. 

**Additional context**
Usually code compiles fine, until you include a file that depends on sdk_config.h in an header, like
#include "opentelemetry/exporters/memory/in_memory_span_exporter.h"

### PR DESCRIPTION
**Pull request by flier**
Fixes #1272

#1273 install `sdk-config.h` to the wrong directory
```
$ tree /usr/local/include/opentelemetry/opentelemetry/
/usr/local/include/opentelemetry/opentelemetry/
└── sdk_config.h

0 directories, 1 file
``` 

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [ ] Changes in public API reviewed

### CODE REVIEW

**Comment by reyang** [comment]
@flier thanks for your contribution! Would you clear the EasyCLA first?

**Review by lalitb — APPROVED** [review]
LGTM once the EasyCLA is fixed.

File: sdk/CMakeLists.txt [hunk 1]
```diff
@@ -14,7 +14,7 @@ install(
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
 install(
-  DIRECTORY include/opentelemetry
+  DIRECTORY include/opentelemetry/
```
[the remark this answers is not in the document]
Comment (lalitb) [reply sdk/CMakeLists.txt:17]: The trailing '/' ensures to copy the contents of source `/include/opentelemetry/*` directory, and not the `/include/opentelemetry` directory itself to destination.

**Review by ThomsonTan — APPROVED** [review]

**Comment by lalitb** [comment]
@flier  - The PR can be merged once the easyCLA signing is fixed. Can you fix that.

**Comment by flier** [comment]
> @flier - The PR can be merged once the easyCLA signing is fixed. Can you fix that.

@lalitb  Sorry, easyCLA has signed.

**Pull request merged**